_assets/javascripts/components/pagination.js

```diff
@@ -83,4 +83,53 @@ CRDS.PageLoader.prototype.activatePage = function(type) {
 
 $(document).ready(function() {
   new CRDS.Pagination();
-});
\ No newline at end of file
+
+  const currentPage = parseInt(paginationInput.getAttribute("placeholder"));
```

Comment:
I'm getting a `Uncaught TypeError: Cannot read property 'getAttribute' of undefined` error I believe is related to `paginationInput` being undefined.

How I would handle something like as follows.
```javascript
$(document).ready(function() {
  const paginationInput = document.querySelector('.pagination-selector');

  if (!paginationInput) {
    return;
  }

  new CRDS.Pagination();

  const currentPage = parseInt(paginationInput.getAttribute("placeholder"));
  // ...
```